awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsApplicationSignalsCustomizerProvider.java

```diff
@@ -221,6 +260,10 @@ private SdkTracerProviderBuilder customizeTracerProviderBuilder(
         return tracerProviderBuilder;
       }
 
+      if (isSigV4Enabled) {
```

Comment:
Not following. Why we pass all the remaining initialization flow if SigV4 is enabled.